src/backend/cuda/include/yaksuri_cudai.h

```diff
@@ -79,10 +79,10 @@ int yaksuri_cudai_md_alloc(yaksi_type_s * type);
 int yaksuri_cudai_populate_pupfns(yaksi_type_s * type);
 
 int yaksuri_cudai_ipack(const void *inbuf, void *outbuf, uintptr_t count, yaksi_type_s * type,
-                        yaksi_info_s * info, int target);
+                        yaksi_info_s * info, yaksa_op_t op, int target);
 int yaksuri_cudai_iunpack(const void *inbuf, void *outbuf, uintptr_t count, yaksi_type_s * type,
-                          yaksi_info_s * info, int target);
-int yaksuri_cudai_pup_is_supported(yaksi_type_s * type, bool * is_supported);
+                          yaksi_info_s * info, yaksa_op_t op, int target);
+int yaksuri_cudai_pup_is_supported(yaksi_type_s * type, yaksa_op_t op, bool * is_supported);
```

Comment:
The commit message seems a bit misleading. This commit has a lot of changes in the backend also.

Reply:
Yeah, the "frontend" tag needs to be removed.

Reply:
Other than that the message itself is fine, right?

Reply:
Yes, if the tag is corrected, the rest seems fine.

Reply:
Fixed.